tests/phpunit/tests/actions/callbacks.php

```diff
@@ -7,6 +7,8 @@ class Tests_Actions_Callbacks extends WP_UnitTestCase {
 
 	/**
 	 * @ticket 23265
+	 *
+	 * @covers ::has_action
```

Comment:
```suggestion
	 * @covers ::add_action
```

Ticket 23265 improved `_wp_filter_build_unique_id` to make it easier for `remove_action` and other hook functions to work with callbacks in different representations. It's part of the callback registration process. Notice that it's also a private function to encapsulate reusable functionality for use within core. As it's private, the covers is applied to the consuming function, i.e. `add_action` in this case.